### ISSUE TITLE
[DevNote] Introduce dedicated function to set autoload value of an option already in the database

### ISSUE THREAD

**Issue by shabnam611** [WordPress/Documentation-Issue-Tracker#1175]
Trac ticket: https://core.trac.wordpress.org/ticket/58964
Keyword: needs-dev-note
Ticket owner: flixos90
Core committer: tabrisrp

**Comment by codente** [comment]
Will be included in Performance Team's cluster "Optimizing autoloaded options dev note"
https://github.com/WordPress/performance/issues/840

**Comment by codente** [comment]
Dev note still in draft at this point

**Comment by nalininsbs** [comment]
@codente and @shabnam611 is this covered by one of the mixed dev notes from performance? Thank you

**Comment by shabnam611** [comment]
@nalininsbs 
Yes, dev note is added here https://make.wordpress.org/core/2023/10/17/new-option-functions-in-6-4/
